src/contribute/contribution-guide.md

```diff
@@ -201,10 +201,12 @@ For contributions to the Java code, run unit tests locally via Maven. Alternativ
 Once the initial code is complete and the tests pass, it’s time to start the code review process. We review and discuss all code, no matter who authors it. It’s a great way to build community, since you can learn from other developers, and they become familiar with your contribution. It also builds a strong project by encouraging a high quality bar and keeping code consistent throughout the project.
 
 ### Create a pull request
-Organize your commits to make your reviewer’s job easier. Use the following command to re-order, squash, edit, or change description of individual commits.
+Organize your commits to make your reviewer’s job easier. Reviewers normally prefer a minimum number of commits that break up a PR into logical steps. For most PRs, you'll squash your changes down to 1 commit. You can use the following command to re-order, squash, edit, or change description of individual commits.
 
     $ git rebase -i origin/master
 
+You'll then push to your branch on github. Note: when updating your commit after PR feedback and use squash to get back to one commit, you will need to do a force submit to the branch on your repo.
```

Comment:
updated github text.

for other text: 
This information about needing to force push is non-trivial to discover for new git users. General guidance given in git documentation is "force pushing is unusual/bad".  

Generaly, if we think it's necessary to provide the git commands to do things, then I think this information is consistent with providing that level of detail. 

If we didn't provide any of the exact commands (implying that we assume our users already know how to use git), then I would agree with you that this is unnecessary info.